.github/workflows/test.yaml

```diff
@@ -10,7 +10,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-
+      - name: Install Tor
+        run: |
+          sudo apt-get install -y tor
+      - uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+      - name: Install Python dependencies
+        working-directory: ./tools
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Generate authority and fresh dir info
+        working-directory: ./tools
+        env:
+          DIR_AUTH_PASSWORD: dummy
```

Comment:
debug?

Reply:
The underlying command `tor-gencert` requires a password to encrypt the authority's identity key. I use this environment variable to pass it to my script. (cf. our discussion in Slack on how to provide the password)

Reply:
But shouldn't this be passed as a [GitHub secret](https://docs.github.com/en/actions/reference/encrypted-secrets)? Or this is used just for tests?